cylc/flow/cfgspec/workflow.py

```diff
@@ -1490,6 +1490,33 @@ def upg(cfg, descr):
         ['scheduling', 'hold after cycle point']
     )
 
+    u.deprecate(
+        '8.0.0',
+        ['runtime', '__MANY__', 'suite state polling'],
+        ['runtime', '__MANY__', 'workflow state polling'],
+    )
```

Comment:
This ought to have be tested in `tests/functional/deprecations/01-cylc8-basic.t`

Reply:
Done.